pages/common/go-test.md

```diff
@@ -0,0 +1,24 @@
+# go test
+
+> Tests Go packages (files have to end with `_test.go`).
+> More information: <https://golang.org/cmd/go/#hdr-Testing_flags>.
+
+- Test the package found in the current directory:
+
+`go test`
+
+- [v]erbosely test the package in the current directory:
+
+`go test -v`
+
+- Test the packages in the current directory and all subdirectories:
+
+`go test -v ./...`
```

Comment:
This example is confusing a bit — is this example basically equivalent (and true)?
```suggestion
- Test the packages in the specified directory and all subdirectories:

`go test -v {{path/to/directory}}`
```

Reply:
The `go test` tool always takes into consideration the current directory, hence why in the first example no argument was required. The `./` part of the argument specifies that the package in the current directory must be tested, and the `...` part relates to all the other subdirectories, recursively.

Reply:
Clarifying, the given examples would not be equivalent. If you pass a specific directory path to the `go test` command, it'll only test the packages found at that directory, therefore ignoring any subdirectory.

Reply:
Oh, I see. So `...` isn't just a placeholder value, you'd actually pass it to the command?

Reply:
Yes, that is correct.

Reply:
Then maybe
```suggestion
- Test the packages in the current directory and all subdirectories (note the `...`):

`go test -v ./...`
```